src/app/geo/geocode/mapzen-geocode.service.spec.ts

```diff
@@ -1,59 +1,112 @@
 /* tslint:disable:no-unused-variable */
-
 import { TestBed, inject } from '@angular/core/testing';
 import { HttpModule, Http, BaseRequestOptions, ResponseOptions, Response } from '@angular/http';
 import { MockBackend } from '@angular/http/testing';
 import 'rxjs/add/observable/of';
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/catch';
+import 'rxjs/add/observable/throw';
 
 import { MapzenGeocodeService } from './mapzen-geocode.service';
-import { MAPZEN_BASE_URL } from '../../../config/mapzen.config';
+import { MAPZEN_SEARCH_URL } from '../../../config/mapzen.config';
 
 describe('MapzenGeocodeService', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [HttpModule],
+      imports: [ HttpModule ],
       providers: [
-        {provide: MAPZEN_BASE_URL, useValue: 'http://veinapp.test.coop'},
+        { provide: MAPZEN_SEARCH_URL, useValue: 'http://veinapp.test.coop' },
         MapzenGeocodeService,
         {
           provide: Http,
           useFactory: (mockBackend, options) => {
             return new Http(mockBackend, options);
           },
-          deps: [MockBackend, BaseRequestOptions]
+          deps: [ MockBackend, BaseRequestOptions ]
         },
         MockBackend,
         BaseRequestOptions
       ]
     });
   });
 
-  it('should return a CurrentSearchState object with address, lat and long attributes',
-    (inject([MapzenGeocodeService, MockBackend], (mapzenService, mockBackend) => {
+  // mocked data
+  const geoData = {
```

Comment:
I think there is no need to be declared every time in any before test, besides that they are not used by all of them.